estate/models/estate_property.py

```diff
@@ -0,0 +1,48 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = 'estate.property'
+    _description = 'Real Estate Property'
+
+    name = fields.Char(string='Name', required=True)
+    description = fields.Text(string='Description')
+    postcode = fields.Char(string='Postcode')
+    date_availability = fields.Date(string='Availability Date', copy=False, default=fields.Datetime.add(fields.Datetime.today(), days=7))
+    expected_price = fields.Float(string='Expected Price', required=True)
+    selling_price = fields.Float(string='Selling Price', readonly=True, copy=False)
+    bedrooms = fields.Integer(string='Bedrooms', default=2)
+    living_area = fields.Integer(string='Living Area (sqm)')
+    facades = fields.Integer(string='Facades')
+    garage = fields.Boolean(string='Garage Available')
+    garden = fields.Boolean(string='Has Garden')
+    garden_area = fields.Integer(string='Garden Area (sqm)')
+    garden_orientation = fields.Selection(
+        [('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')],
+        string='Garden Orientation'
+    )
+
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        [('new', 'New'), ('offer received', 'Offer Received'), ('offer accepted', 'Offer Accepted'), ('sold', 'Sold'), ('cancelled', 'Cancelled')],
+        default='new',
+        copy=False,
+        required=True,
+    )
```

Comment:
```suggestion
    state = fields.Selection([
	("new", "New"), 
	("offer_received", "Offer Received"), 
	("offer_accepted", "Offer Accepted"), 
	("sold", "Sold"), 
	("canceled", "Canceled")
], string="State", required=True, copy=False, default="new")
```
just a matter of coding style. to gain some lines without being unreadable :)
also, don't use white spaces for the technical name of your selection options.